activerecord/lib/active_record/validations/uniqueness.rb

```diff
@@ -8,6 +8,10 @@ def initialize(options)
           raise ArgumentError, "#{options[:conditions]} was passed as :conditions but is not callable. " \
                                "Pass a callable instead: `conditions: -> { where(approved: true) }`"
         end
+        unless Array(options[:scope]).all? { |scope| scope.respond_to?(:to_sym) }
+          raise ArgumentError, "scope option of the uniqueness validator was passed in unknown format. " \
+            "Pass a symbol or an array of symbols instead: `scope: :user_id`"
```

Comment:
I think we should tell which scope option is incorrect like we did with `:conditions`